GliaWidgets/Sources/EntryWidget/EntryWidget.swift

```diff
@@ -20,11 +20,23 @@ public final class EntryWidget: NSObject {
         dividerHeight: 1
     )
 
-    init(theme: Theme) {
-        self.theme = theme
+    private let environment: Environment
+
+    private var cancellables = CancelBag()
+
+    init(environment: Environment) {
+        self.environment = environment
+        super.init()
+
+        environment.queuesMonitor.$state
+            .sink(receiveValue: handleQueuesMonitorUpdates(state:))
+            .store(in: &cancellables)
     }
 
     public func show(by presentation: EntryWidget.Presentation) {
```

Comment:
Please add documentation for public methods and class itself

Reply:
The documentation should be added along with UI completion by @rasmustautsglia . I was only integrating queues observation logic.